apps/nextra/pages/en/build/create-aptos-dapp/templates/_meta.tsx

```diff
@@ -2,6 +2,9 @@ export default {
   boilerplate: {
     title: "Boilerplate Template",
   },
+  "clicker-game-tg-mini-app": {
```

Comment:
This isn't correct. It needs to be the name of the file otherwise you get two in the sidebar
![Screenshot 2024-09-09 at 4 33 56 PM](https://github.com/user-attachments/assets/8132e2d0-c7ea-4599-9e54-b68453397310)